### DISCUSSION TITLE
Does T-POT support rollback to clear the malicious artifacts from attacker periodically?

### DISCUSSION THREAD

**Discussion by PinGMUICT** [telekom-security/tpotce#1420] [General]
If an attacker places malicious file on my t-pot and uses it to attack others. Can attacker do?
I am not sure if t-pot has any mechanism to clean all malicious artifacts periodically or not?

Thanks in advance.

**Comment by t3chn0m4g3** [comment]
By default all logs and artifacts are rotated on a daily basis. And all containers are started freshly from their respective image.
